tests/integrationv2/common.py

```diff
@@ -403,6 +403,7 @@ def __init__(self,
             verify_hostname=None,
             server_name=None,
             protocol=None,
+            older_version=None,
             env_overrides=dict()):
```

Comment:
I don't love this name. Minimally it should probably be "use_older_version", but maybe "use_mainline_version" instead?

Reply:
Yeah, that's a way better name